lib/scrabble/player.rb

```diff
@@ -0,0 +1,122 @@
+module Scrabble
+  class Player
+    attr_reader :name, :plays, :tiles
+
+
+    ## -------------------------------------------------------------------------
+    # CLASS METHODS
+
+
+    def initialize(name)
```

Comment:
Yeah - Foo.new calls Foo#initialize behind the scenes - it's a ruby thing.
"method that sets up an instance" is more generally called "constructor" in most programming languages.